### DISCUSSION TITLE
如何实现pdf论文总结

### DISCUSSION THREAD

**Discussion by pengzy-cosmo** [lobehub/lobehub#4091] [General Question]
对RAG的原理不太了解，我想实现对某一篇pdf格式的论文实现全文分析和总结。但是上传分块后，模型一次只会读取其中几块，而且不会意识到这是一篇完整论文。我想问一下，如何实现像市面上那些pdf总结工具一样，一次对某篇文章进行全文分析

**Comment by SpeedupMaster** [comment]
现在还没做，等后续更新 #4005
